wallet_ui/utils/authClientLegacy.js

```diff
@@ -0,0 +1,274 @@
+import {
```

Comment:
Is this being used anywhere?

Reply:
No it's not, I thought I deleted the file. Will change it.